.eslintrc.json

```diff
@@ -19,7 +19,8 @@
     },
     "plugins": ["react", "@typescript-eslint", "unused-imports"],
     "rules": {
-        "unused-imports/no-unused-imports": "error"
+        "unused-imports/no-unused-imports": "error",
+	"react/display-name": "warn"
```

Comment:
it's still sorta weird that we're getting this as an error, since the items it is warning on are not react components they are normal functions.

Reply:
hmm. From my limited react days from '16 iirc there are "pure components" that are just functions. Maybe they shipped a bug in how pure components are identified?

Reply:
wouldn't surprise me, since some of these are literally just functions, not even "pure components" (e.g., return type is not a React.Component). For now, warn is OK.